src/11_args.py

```diff
@@ -6,13 +6,25 @@
 
 # YOUR CODE HERE
 
+def f1(num1, num2):
+    return num1 + num2
+
 print(f1(1, 2))
 
 # Write a function f2 that takes any number of integer arguments and returns the
 # sum.
 # Note: Google for "python arbitrary arguments" and look for "*args"
 
 # YOUR CODE HERE
+def f2(*args):
+  count = 0
+  
+  for num in args:
+      if type(num) != list:
+          count += num
+      else:
+          count = sum(num)
+  return count
 
 print(f2(1))                    # Should print 1
 print(f2(1, 3))                 # Should print 4
```

Comment:
Changed the example data and didn't write the function correctly. The function should take in an arbitrary number of integers as its argument. If you want to pass a list, you should unpack the list first.